semantics/language/translation/decl/tagged.k

```diff
@@ -36,7 +37,7 @@ module C-DECL-TAGGED
           => addTag(X, makeUnionFieldInfo(L))
```

Comment:
I've changed these rules to reflect the error more accurately. Lets keep the behavior as undefined for now.